### ISSUE TITLE
[Downloader V3] Accessing files bundled with cogs

### ISSUE THREAD

**Issue by Aioxas** [Cog-Creators/Red-DiscordBot#1040]
In V2, we could go from whenever red.py was to the cog folder which was essentially "/cogs/spoiler" or to the data folder "data/spoiler" and then work from there.

This is not the case for v3. An example is shown below.

Ex.: https://github.com/flapjax/FlapJack-Cogs/tree/red-v3-rewrites/spoiler
The cog breaks because the location of cwd is wherever we ran redbot <instance name> <flags>
so we would have to find the location using the code below.

After doing that, we can work with the bundled data folder or create new files or anything. But right now that's something that is not explained. Whether this will be fixed in a future revision of Downloader or something, I wanted to point this out for current iterations of Downloader & Red V3.

TL;DR: Files that come bundled with the .py file have to be accessed using a special method that involves finding the cog's location, changing the current working directory to that and working from there.

```py

path = os.path.dirname(os.path.realpath(__file__))
os.chdir(path)
dir_path = os.getcwd()

**Comment by tekulvw** [comment]
Fixed with #1063
